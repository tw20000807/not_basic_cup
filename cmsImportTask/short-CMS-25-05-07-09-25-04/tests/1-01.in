SX4sl1lxDFZr39zzEnirMRn9hy1cLOWa
3
1 2 3