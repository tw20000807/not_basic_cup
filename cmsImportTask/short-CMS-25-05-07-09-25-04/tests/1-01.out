gCf9Urk7brY9n0BAGoLGC2RnqQx0ZL06
OK
2 3 1
